### ISSUE TITLE
Change "main" indentifier

### ISSUE THREAD

**Issue by benjamingeorge** [airbnb/react-sketchapp#147]
I'm have a style-guide app.  I want to drop in react-sketchapp to render to sketch.  I noticed that in package.json, the "main" property has end in ".sketchplugin" in order to work.  is there a way to use a different identifier property so I can keep my "main" : "index.js" ?  I tried changing the identifier in manifest.json but this didn't work.  

If not, do I have to maintain a separate sketchplugin package that has my style-guide as a dependency? What are some best practices ?

**Comment by mathieudutour** [comment]
There is no way to do that with skpm for now. But we do have a PR opened for it: https://github.com/skpm/skpm/pull/44

**Comment by jongold** [comment]
Closing this as it's a limitation with skpm right now. Indeed we've been using separate modules to get around this — you could also use webpack without skpm by digging around through skpm's source :)

**Comment by mathieudutour** [comment]
I just published [skpm@0.10.1](https://github.com/skpm/skpm/releases/tag/v0.10.1) which fixes this.

In your package.json you can now write:
```
{
   ...
  "main": "index.js",
  "skpm": {
    "main": "plugin.sketchplugin",
    "manifest":...
  }
}
```

**Comment by benjamingeorge** [comment]
Great thanks!
